site/en/docs/crux/methodology/index.md

```diff
@@ -0,0 +1,565 @@
+<a id="tooltip-2"><sup>2</sup></a>: 28-day rolling average data is updated daily, based on the aggregated data from the previous 28 days.<br>
+<a id="tooltip-3"><sup>3</sup></a>: Not all metrics are available in all monthly tables, see the <a href="../release-notes">release notes</a> for details.<br>
+<a id="tooltip-4"><sup>4</sup></a>: The web vital metrics are available in all tools.<br>
+</p>
+
+The following sections briefly summarize each tool and how the data can be used.
+
+### CrUX on BigQuery {: #tool-bigquery}
+
+Origin-level CrUX data is available for public querying via [BigQuery](https://cloud.google.com/bigquery). Read the guide on [Using the Chrome UX Report](https://web.dev/chrome-ux-report-bigquery/).
+
+[CrUX on BigQuery](../bigquery/) provides a publicly accessible database of all origin-level data collected by CrUX. It is possible to query any and all origins for which data is collected, analyze any metric that CrUX supports and filter by all available dimensions. Full metric histograms are stored in the BigQuery tables allowing for visualization of performance distributions, including experimental metrics.
+
+The data in BigQuery is updated monthly, with each month's data released on the second Tuesday after the collection period. Page-level data is not available in BigQuery tables, and percentiles are interpreted from coarse histogram data which results in approximate values.
+
+Use CrUX on BigQuery for analysis across any dimension: origins, countries, dates, form factor and connection type. Read more on the [CrUX on BigQuery](../bigquery/) documentation page.
+
+### CrUX Dashboard {: #tool-crux-dash}
+
+The [CrUX Dashboard](https://web.dev/chrome-ux-report-data-studio-dashboard/) is a Data Studio dashboard that allows you to query and render CrUX data into an interactive dashboard, as well as exporting PDF reports.
+
+The dashboard provides visualization of all CrUX metrics in monthly trends, with data available back to 2017. Data can be split by form factor to compare mobile / tablet / desktop performance and performance goals are available to create red-amber-green visualizations.
+
+The CrUX dashboard does not support effective connection type or country dimensions, so all global data is presented in the reports. Page-level data is not available and percentile values are calculated from coarse histogram data so are approximate.
+
+### CrUX API {: #tool-crux-api}
+
+The [CrUX API](https://web.dev/chrome-ux-report-api/) provides programmatic access to CrUX data by page or origin.
+
+The API provides web vitals metrics both by origin and at page-level and the data is updated daily. The only values provided for metrics are calculated from the previous 28 days as a rolling window, no historical data is available via the API.
+
+[Read more in the API documentation](/crux/docs/api/).
```

Comment:
The distinction between the web.dev / DCC pages should be: "how to use it" on web.dev, and "official product docs" on DCC. I'm open to whatever works best for consistency and clarity. Agreed that updating the first link so that it goes to the product page or documentation page makes more sense. We could still link to the web.dev guides while clarifying that this is where readers can learn more about these tools.